be/src/exec/vectorized/join_hash_map.h

```diff
@@ -489,7 +489,7 @@ class JoinHashMapForEmpty {
                             NullableColumn::create(std::move(column), NullColumn::create(_probe_state->count));
                     (*chunk)->append_column(std::move(dest_column), slot->id());
                 } else {
-                    DCHECK_EQ(column->is_nullable(), to_nullable);
+                    // DCHECK_EQ(column->is_nullable(), to_nullable);
```

Comment:
why we comment this check ?

Reply:
Current planner would generate a wrong nullable property for some special join scenario, which I have not figured out. So I choose to make the join executor support both `nullable` and `not-nullable`.